change/@ni-nimble-angular-1fa32c6c-a6b6-424d-a969-eccb45a9b0f1.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Changes product code so I think should still be a patch. Even a comment could have unexpected side-effects from funky build tooling, etc.

```suggestion
  "type": "patch",
```